# noncoding_visualisations ðŸ§¬
A collection (work in progress) of tools to help with visualisation of the non-coding, regulatory genome.

## Genome browsers

### [Epilogos](https://epilogos.altius.org/)

Features:
* Shows an information-content-aware visualisation of multi-sample chromatin state data, analogous to how motif logos work
* The chromatin states are derived using tools such as [ChromHMM](http://compbio.mit.edu/ChromHMM/) and [Segway](https://segway.hoffmanlab.org/)
* Pairwise epilogos compare the chromatin state data between groups

Links
* [Browser and documentation](https://epilogos.altius.org/)
* (Manuscript in preparation)
* [Source code and documentation](https://github.com/meuleman/epilogos)

### [WashU Epigenome browser](http://epigenomegateway.wustl.edu/browser/)

Features:
[add]

Links
* [Browser](http://epigenomegateway.wustl.edu/browser/)
* [Manuscript](https://doi.org/10.1093/nar/gkac238) (there are more manuscripts for different updates)
* [Documentation](https://eg.readthedocs.io/en/latest/)
* [Source code](https://github.com/lidaof/eg-react)

### Special mention: dynseq tracks

[add]

## Sequence logo visualisation

### Logomaker (python)

Features:
* Visualise sequence logos, renders as vector graphics
* Integrates with matplotlib axes
* Customise [add]
* [add]

[Documentation](https://logomaker.readthedocs.io/en/latest/)
[Crossref](https://doi.org/10.1093/bioinformatics/btz921)

### ggseqlogo (R)

Features:
* Visualise sequence logos
* Extends ggplot
* [add]



## Hi-C visualisation

[add]
